lms/djangoapps/verify_student/models.py

```diff
@@ -1180,6 +1237,9 @@ def get_location_id(cls, photo_verification):
             return ""
 
 
+# DEPRECATED: this feature has been permanently enabled.
+# Once the application code has been updated in production,
+# this table can be safely deleted.
```

Comment:
Note to reviewers: Now that we're allowing course staff to add ICRV components to courseware, I figured it made sense to permanently enable ICRV end-points.  In other words, this is one less thing we need to remember to configure :)